extensions/wikia/AssetsManager/tests/AssetsManagerSassBuilderTest.php

```diff
@@ -9,30 +9,28 @@ class AssetsManagerSassBuilderTest extends WikiaBaseTest {
 
 	public function setUp() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "setUp" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)

[project-id]: wikia:app